ProyectoFinal/src/visual/ListarEmpresas.java

```diff
@@ -122,9 +124,34 @@ public void mouseClicked(MouseEvent e) {
 			panelFilter.add(btnFilter);
 
 			JButton btnReset = new JButton("Mostrar todas las empresas");
+			btnReset.addActionListener(new ActionListener() {
+				public void actionPerformed(ActionEvent e) {
+					txtRNC.setText("");
+					loadRowsInTable(BolsaTrabajo.getInstance().getEmpresasByID(""), null);
+					btnReset.setEnabled(false);
+				}
+			});
 			btnReset.setEnabled(false);
 			btnReset.setBounds(658, 17, 211, 23);
 			panelFilter.add(btnReset);
+			
+			btnFilter.addActionListener(new ActionListener() {
+				public void actionPerformed(ActionEvent e) {
+					if(Utils.isMaskCedulaDefaultValue(txtRNC.getText())) {
+						JOptionPane.showMessageDialog(null, "Tiene que completar la c�dula", "Advertencia", JOptionPane.WARNING_MESSAGE);
```

Comment:
```suggestion
						JOptionPane.showMessageDialog(null, "Tiene que completar la c\u00e9dula.", "Advertencia", JOptionPane.WARNING_MESSAGE);
```

Para que no de un símbolo extraño por el _encoding_.